_saas-integrations/closeio/v1/closeio-v1.md

```diff
@@ -0,0 +1,90 @@
+---
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: closeio
+permalink: /integrations/saas/closeio
+tags: [saas_integrations]
+keywords: closeio, integration, schema, etl closeio, closeio etl, closeio schema
+layout: singer
+# input: false
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "closeio"
+display_name: "Close.io"
+
+singer: true
+tap-name: ""
+repo-url: https://github.com/singer-io/tap-closeio
+
+# this-version: ""
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+status: "Alpha"
```

Comment:
Metadocs have possible statuses as:

- Closed Beta
- Open Beta
- Released
- Deprecated

Are you proposing we add "Alpha" as well?